src/params/labels.json

```diff
@@ -0,0 +1,11 @@
+{
+    "test": {
+        "simple": "simple value",
+        "complex": "complex {1}"
+    },
+    "footer": {
```

Comment:
way footer label so far from footer declaration, may be organize thing by functional way will be simpler.

Reply:
Now, you have to pass a labels object each time you call getLabel. This labels object can come from a conf file from the component.

Reply:
I do not understand your sentence ^^ :p

Reply:
Now, you have to write a conf file (with the labels) for each component.